Makefile

```diff
@@ -39,7 +39,7 @@ build:
 
 core_api_gen:
     ifeq (, $(shell which oapi-codegen))
-	go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@latest
+	go install github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen@latest
```

Comment:
Can we pin this to a version instead of latest. Julian raised concerns of formatting error here 

https://github.com/astronomer/astro-cli/pull/1550/files#r1489665733